<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOfflineMsgView</class>
 <widget class="QDialog" name="DialogOfflineMsgView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="LBL_DLG_POST_OFFMSGS_TITLE">Offline Message</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lh3.qrc">
    <normaloff>:/lh3/Resources/message_new.png</normaloff>:/lh3/Resources/message_new.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>5</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lCaptionTitle_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string comment="LBL_DLG_POST_OFFMSGS_SUBJECT">Subject:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lSubject">
       <property name="text">
        <string>The test message</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lCpationAuthor_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string comment="LBL_DLG_POST_OFFMSGS_VISITOR">Visitor:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lAuthor">
       <property name="text">
        <string>jessica</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lCaptionDate_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string comment="LBL_DLG_POST_OFFMSGS_EMAIL">Email:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lEmail">
       <property name="text">
        <string>john@example.com</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string comment="LBL_DLG_POST_OFFMSGS_DATE">Date:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="lDate">
       <property name="text">
        <string>10.06.2009 10:51</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="ptMessage">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>5</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string comment="LBL_DLG_POST_OFFMSGS_REPLIED">Replied:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lReplied">
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lRespondent">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string comment="LBL_DLG_POST_OFFMSGS_REPLY_DATE">Reply Date:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lReplyDate">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string comment="LBL_DLG_POST_OFFMSGS_RESPONDENT">Respondent:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="ptReply"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbReply">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string comment="LBL_DLG_POST_OFFMSGS_POST_REPLY">Post Reply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbMarkAsUnread">
       <property name="text">
        <string comment="LBL_DLG_POST_OFFMSGS_MARK_UNREAD">Mark as unread</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string comment="LBL_CLOSE">Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lh3.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogOfflineMsgView</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbReply</sender>
   <signal>clicked()</signal>
   <receiver>DialogOfflineMsgView</receiver>
   <slot>reply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ptReply</sender>
   <signal>textChanged()</signal>
   <receiver>DialogOfflineMsgView</receiver>
   <slot>textChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>-4</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>reply()</slot>
  <slot>replyChanged()</slot>
  <slot>textChanged()</slot>
 </slots>
</ui>
